doc/source/examples.rst

```diff
@@ -140,6 +140,18 @@ You can run these tests from the nagios master or in the target host with
 `NRPE <https://en.wikipedia.org/wiki/Nagios#Nagios_Remote_Plugin_Executor>`_.
 
 
+Integration with KitchenCI
+~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+KitchenCI (aka Test Kitchen) can use testinfra via its :code:`shell` verifier. Just
+copy `conftest.py <conftest.py>`_ into your :code:`test` folder and set the verifier
+in your :code:`.kitchen.yml`::
+
+    verifier:
+      name: shell
+      command: testinfra -vvv --junit-xml "junit-${KITCHEN_INSTANCE}.xml" "test/integration/${KITCHEN_SUITE}"
```

Comment:
Admittedly I do not fully understand how `conftest.py` works so I’m curious if this will have worse performance. On the other hand this will allow for parallel testing of multiple instances I guess.